src/person.rb

```diff
@@ -1,5 +1,10 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
+  # rubocop:disable Style/OptionalBooleanParameter
   def initialize(age, name = 'Unknown', parent_permission = true)
+    # rubocop:enable Style/OptionalBooleanParameter
```

Comment:
- Please rmoeve all disabled linters, remember that linters help us to write code that is clean and easy to read, please fix this 🙏